src/test/resources/expected_noVEPinheritance.vcf

```diff
@@ -0,0 +1,51 @@
+##fileformat=VCFv4.2
+##FILTER=<ID=PASS,Description="All filters passed">
+##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth (reads with MQ=255 or with bad mates are filtered)">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+##FORMAT=<ID=VI,Number=.,Type=String,Description="An enumeration of possible inheritance modes.">
+##FORMAT=<ID=VIC,Number=1,Type=String,Description="Inheritance Compound status.">
+##FORMAT=<ID=VID,Number=1,Type=Integer,Description="Inheritance Denovo status.">
+##FORMAT=<ID=VIG,Number=.,Type=String,Description="Genes with an inheritance match.">
+##FORMAT=<ID=VIM,Number=1,Type=Integer,Description="Inheritance Match status.">
+##FORMAT=<ID=VIS,Number=.,Type=String,Description="An enumeration of possible sub inheritance modes like e.g. compound, non penetrance.">
+##INFO=<ID=CSQ,Number=.,Type=String,Description="Consequence annotations from Ensembl VEP. Format: Allele|SYMBOL|Gene">
+##INFO=<ID=Compounds,Number=.,Type=String,Description="List of compound pairs for this variant.The list is splitted on ',' family id is separated with compoundswith ':'. Compounds are separated with '|'.">
+##INFO=<ID=GeneticModels,Number=.,Type=String,Description="':'-separated list of genetic models for this variant.">
+##INFO=<ID=ModelScore,Number=.,Type=String,Description="PHRED score for genotype models.">
+##PREFERRED=Flags a transcript as preferred
```

Comment:
Suggestion: remove all references to PREFERRED transcripts (plugin will be removed in VIP v4.0.0)